terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfView.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (C) 2013-2018 NTT DATA Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+package org.terasoluna.tourreservation.app.managereservation;
+
+import java.text.SimpleDateFormat;
+import java.util.Map;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.web.servlet.view.document.AbstractPdfStamperView;
+
+import com.lowagie.text.pdf.AcroFields;
+
+@Component
+public class ReservationReportPdfView extends AbstractPdfStamperView {
+
+    @Value("${reservation.report.pdf.path}")
+    String reservationReportPdfPath;
```

Comment:
- [ ] Good to name not `Path` but `Url`. Because you set actual URL formed value ex, `classpath:xxx/yyy` `file:xxx/yyy`.

Reply:
f777cb00e99c8a30dcb912a633a3cfaf7d1dad40

I changed Path -> Url in above commit .